src/Views/ValorantTeam.jsx

```diff
@@ -31,19 +32,19 @@ const ValorantTeam = () => {
       </div>
       <div className="l-container">
         {/* FOR MAP */}
-        {/* {teams.map((team) => (
-          <div className="l-container" key={team.teamLogo}>
+        {teams.map((team) => (
+          <div className="l-container" key={team.teamLogoURL}>
             <Link to="/team">
-              <Astrateamcards teamlogo={team.teamLogo} tname="Millennia" />
+              <Astrateamcards teamlogo={team.teamLogoURL} tname="Millennia" />
```

Comment:
I guess this can now be changed as Dynamic data since we now have a dynamic team name

tname="Millennia"